app/components/features/features.styl

```diff
@@ -31,10 +32,12 @@
       -webkit-padding-start 30px
       -moz-padding-start 30px
     .content-wrap
-      width 40%
+      width: 376px;
```

Comment:
You suggest to hardcode the feature card width? What if 376px will not fit to the screen?